src/aircrafts/F16/propulsion.jl

```diff
@@ -6,7 +6,7 @@ import FlightMechanics.Models:
     get_engine_position, get_engine_orientation, get_engine_gyro_effects,
     calculate_engine
 
-export F16Engine
+export F16Engine,
```

Comment:
Good point. Was that supposed to work before?

Reply:
it was working before. but when i tried add other function in export then it shows error so better to just add this.